mava/networks.py

```diff
@@ -142,7 +147,13 @@ def __call__(self, obs_embedding: chex.Array, observation: Observation) -> tfd.I
             tfd.Independent: Independent transformed distribution.
         """
         loc = self.mean(obs_embedding)
-        scale = jax.nn.softplus(self.log_std) + self.min_scale
+
+        if self.independent_std:
+            scale = jax.nn.softplus(self.log_std) + self.min_scale
+        else:
+            scale = self.log_std(obs_embedding)
+            scale = jax.nn.softplus(scale) + self.min_scale
```

Comment:
NIT:
``` suggestion
        if self.independent_std:
            scale = self.log_std
        else:
            scale = self.log_std(obs_embedding)
      scale = jax.nn.softplus(scale) + self.min_scale
``` 

Shorter but not sure if readable:
``` python
     scale = self.log_std if self.independent_std else self.log_std(obs_embedding)
     scale = jax.nn.softplus(scale) + self.min_scale

```

Reply:
^ like this!